apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js

```diff
@@ -78,7 +78,7 @@ describe('The JSDebugger component', () => {
   });
 
   it('initially has the height of 120px', () => {
-    expect(debugAreaEl().style.height).to.equal('120px');
+    expect(debugAreaEl().props().style.height).to.equal(120);
```

Comment:
This is changing what's tested: from checking if it's actually the right height, to verifying a prop is passed in. Intentional?

Reply:
Will fix